src/instructlab/sdg/generate_data.py

```diff
@@ -365,6 +540,38 @@ def generate_data(
         logger.info("Generated %d samples" % len(generated_data))
         logger.debug("Generated data: %s" % generated_data)
 
+        if is_knowledge:
+            knowledge_phase_data = _create_phase07_ds(logger, new_generated_data)
+            output_file_leaf_knowledge = (
+                f"node_datasets_{date_suffix}/node_{i}_p07.jsonl"
```

Comment:
I'm missing context here but I'd love to know the signifigance of p07 and p10 below

Reply:
I don't have full context on why they're named this, but did push some additional changes that attempts to document what's in these p07 and p10 datasets at least.